### PR TITLE
Add `/v1/breach` endpoint support on `user-intel` service.

### PR DESCRIPTION
**Pull request by pangea-andrest**

### CODE REVIEW

File: packages/pangea-sdk/pangea/asyncio/services/intel.py [hunk 1]
```diff
@@ -985,3 +985,41 @@ async def password_breached_bulk(
...
+                breach_id="66111",
+            )
+        """
+
+        input = m.BreachedRequest(
+            breach_id=breach_id,
+            provider=provider,
+            verbose=verbose,
+        )
+        return await self.request.post("v1/breached", m.BreachedResult, data=input.model_dump(exclude_none=True))
```
Comment (kenany) [inline packages/pangea-sdk/pangea/asyncio/services/intel.py:1025]: In LVE I see `v1/breach`.

**Review by kenany — CHANGES_REQUESTED** [review]
Still has some minor review comments and now a merge conflict.

**Comment by pangea-andrest** [comment]
@kenany Updated!

File: CHANGELOG.md [hunk 2]
```diff
@@ -18,8 +18,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- Support for `cursor` field on `v1/user/breached` of `user-intel`
+- Support for `cursor` field on `v1/user/breached` of `user-intel` service.
 - `vault_parameters` and `llm_request` fields support on Redact service.
+- Support for `severity` field on `v1/user/breached` and `v2/user/breached` of `user-intel` service.
+- `/v1/breach` endpoint support on `user-intel` service.
```
Comment (kenany) [inline CHANGELOG.md:24]: Need to go under `## Unreleased` now.

**Review by kenany — CHANGES_REQUESTED** [review]

**Review by kenany — APPROVED** [review]

**Pull request merged**